### PR TITLE
Add port option to allow multiple instances of the gnirehtet relay to run 

### PR DESCRIPTION
**Pull request by jessedusty**
Related to https://github.com/Genymobile/gnirehtet/issues/179 but I have a use case for running multiple instances. 

In my use case, I need to be able to send traffic to different proxies for different phones. I currently run gnirehtet with proxychains to do this. This added -p option lets me run multiple instances of the relay on the same computer. 

It was useful for what I'm doing so I thought I'd try to get it upstream to see if it helps anyone else.

I added the option to both the Java and Rust versions to keep the versions consistent. I'm not super familiar with Rust so if there are any conventions I missed, let me know.

### CODE REVIEW

File: relay-java/src/main/java/com/genymobile/gnirehtet/relay/Relay.java [hunk 1]
```diff
@@ -25,7 +25,7 @@
 
     private static final String TAG = Relay.class.getSimpleName();
 
-    private static final int DEFAULT_PORT = 31416;
+    public static final int DEFAULT_PORT = 31416;
```
Comment (jessedusty) [inline relay-java/src/main/java/com/genymobile/gnirehtet/relay/Relay.java]: Just so I can access the default port in the command line argument class

**Comment by ars2005tron** [comment]
Hello, enjoyed your program version of the 2.1 ! the program has changed in Android Studio! added activiti with input fields IP and DNS, the Program worked badly, constantly stops. My change works well! IP which I enter the interface on Android as a result of check of WebRTC show the IP necessary to me. Please add a menu to gnirehtet.APK-enter the IP address of the interface to connect and preferably enter DNS! Thanks.

**Comment by imonlygitting** [comment]
very useful indeed, is this something that can be done  automatically or do we need to find the port of attached device and input at command line? what if the ports change (eg due to plugging into a different usb socket)?

**Comment by rom1v** [comment]
Thank you, I applied some minor changed and merged the commits into `master`:
 - 308d850f457270a1ec1423a57874efd1eddcf5b6
 - a84a4d9e44b8970abb1bf958452fad5dcba8e13c

**Pull request closed without merging**